ecommerce_integrations/unicommerce/order.py

```diff
@@ -386,3 +395,14 @@ def _get_warehouse_allocations(sales_order):
 			}
 		)
 	return item_details
+
+
+@frappe.whitelist()
+def resync_item(code):
+	from ecommerce_integrations.unicommerce.utils import force_sync
+
+	id_ecommerce = frappe.get_value("Ecommerce Item", {"erpnext_item_code": code})
+	if id_ecommerce:
+		frappe.delete_doc("Ecommerce Item", id_ecommerce)
+		force_sync(document="Items")
+	return
```

Comment:
If it's about ecommerce item it should be in `product.py`?

Reply:
The item isn't actually deleted from unicommerce so how does resync fix anything?

Reply:
Removed